src/system_metrics/mongoose_system_metrics_collector.erl

```diff
@@ -215,3 +216,12 @@ calculate_stanza_rate(PrevReport, NewCount) ->
             undefined -> Count;
             Total -> Count-Total
         end} || {Type, Count} <- NewCount].
+
+get_config_type() ->
+    ConfigPath = ejabberd_config:get_config_path(),
+    ConfigType = case filename:extension(ConfigPath) of
+        ".toml" -> toml;
+        ".cfg" -> cfg;
+        _ -> unknown_config_type
```

Comment:
This is not reachable - `mongoose_config_parser:parse_file` is called before and it would crash.

Reply:
Sure, I just wanted to be double cautious just in case. If you think line 225 is unnecessary, I'll have it removed.

Reply:
The case expression in `mongoose_config_parser:parse_file` has only two clauses and I think these two should be consistent as they are checking the same thing. Services are started after the config is parsed, so anything other than `.toml` or `.cfg` here would mean a bug in the code.

Reply:
I hoped we could remove that line, but it is minor, so I am proceeding with the merge.